src/ebonite/client/base.py

```diff
@@ -153,31 +153,35 @@ def get_environment(self, name: str) -> RuntimeEnvironment:
         """
         return self.meta_repo.get_environment_by_name(name)
 
-    def run_service(self, name: str, image: Image, environment: RuntimeEnvironment = None, **kwargs) -> RuntimeInstance:
+    def run_service(self, name: str, image: Image, environment: RuntimeEnvironment = None,
+                    service_type: str = 'docker', **kwargs) -> RuntimeInstance:
```

Comment:
why would you need to pass `service_type` if it can be derived from `image` type?
or maybe from env type?

Reply:
It couldn't be derived from `image` type as we could (in future) run Docker images as either Docker container or Kubernetes pod.

It potentially could be derived from `environment` type (if every `instance` type has the only `environment` type and vice versa, and for now I'm not sure about it) but this argument is optional.

In any case we support just Docker for now so there is no need to specify anything really. I suppose to wait for UI/API implementation and support for Kubernetes (or smth else) to discover whether we need to improve usability here and if so how do we need to do it.